pom.xml

```diff
@@ -20,19 +21,29 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.hibernate</groupId>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.12.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.mysql</groupId>
+            <artifactId>mysql-connector-j</artifactId>
+            <version>8.0.32</version>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate.orm</groupId>
             <artifactId>hibernate-core</artifactId>
-            <version>5.4.21.Final</version>
+            <version>6.2.0.CR3</version>
         </dependency>
         <dependency>
-            <groupId>org.hibernate</groupId>
+            <groupId>org.hibernate.orm</groupId>
             <artifactId>hibernate-entitymanager</artifactId>
-            <version>5.4.21.Final</version>
+            <version>6.0.0.Alpha7</version>
```

Comment:
Use alpha version is a bad practice, cause: 
Using alpha versions of dependencies in Maven can be risky as these versions are typically still in development and have not been thoroughly tested. Alpha versions may contain bugs or other issues that could cause instability in your project or lead to unexpected behavior.